docs/testing/debug-add-ins-overview.md

```diff
@@ -27,15 +27,28 @@ Debugging the server-side code of an Office Add-in is the same as debugging the
 
 The rest of this article is concerned only with debugging client-side JavaScript (which may be transpiled from TypeScript).
 
+## Special cases
+
+There are some special cases in which the debugging process is different from what it normally is for a given combination of platform, Office application, and development environment. If you are debugging any of these special cases, use the links in this section to find the proper guidance. Otherwise, continue to [General guidance](#general-guidance).
```

Comment:
```suggestion
There are some special cases in which the debugging process is different from what it normally is for a given combination of platform, Office application, and development environment. If you're debugging any of these special cases, use the links in this section to find the proper guidance. Otherwise, continue to [General guidance](#general-guidance).
```